packages/config/config/devices/0x045a/Z-CM-V01.json

```
@@ -0,0 +1,218 @@
+{
+	"manufacturer": "ZVIDAR",
+	"manufacturerId": "0x045a",
+	"label": "Z-CM-V01",
+	"description": "Smart Curtain Motor",
+	"devices": [
+		{
+			"productType": "0x0904",
+			"productId": "0x0507"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Set To Start Holding Hands",
+			"description": "This parameter can be used to set the motor open hand start function",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Close hand start function",
+					"value": 0
+				},
+				{
+					"label": "Open hand start function",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Set Motor Direction",
+			"description": "This parameter can be used to set the motor rotation direction.",
```

```suggestion
			"label": "Motor Direction",
```
We prefer short labels and omit descriptions if they are not absolutely necessary.